### ISSUE TITLE
Can't import aws_elastictranscoder_preset if description is missing

### ISSUE THREAD

**Issue by robstar** [terraform-providers/terraform-provider-aws#10758]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

terraform -v
Terraform v0.12.13
+ provider.aws v2.34.0

### Affected Resource(s)

* aws_elastictranscoder_preset

### Terraform Configuration Files

```hclresource 
"aws_elastictranscoder_preset" "my_preset" {
}
```

### Panic Output

```Error: rpc error: code = Unavailable desc = transport is closing

panic: runtime error: invalid memory address or nil pointer dereference
```

### Expected Behavior

The resource has been created in the web interface and should be imported into Terraform even if the (optional) parameter "description" is missing/empty.

### Actual Behavior

panic: runtime error: invalid memory address or nil pointer dereference

### Steps to Reproduce

1. `terraform apply`
`terraform import module.my_module.aws_elastictranscoder_preset.720p 1234567890123456-abc123`

**Comment by bflad** [comment]
Hi @robstar 👋 Sorry you ran into this. Could you please post the next few panic stacktrace lines under `panic: runtime error: invalid memory address or nil pointer dereference`? This has the exact code location that caused the panic. Thanks so much.

**Comment by robstar** [comment]
Of course, sorry for missing :see_no_evil: 
```[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x34113e5]
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: 
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: goroutine 106 [running]:
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: github.com/terraform-providers/terraform-provider-aws/aws.resourceAwsElasticTranscoderPresetRead(0xc0001c4620, 0x3c8e740, 0xc00055fb00, 0xc0001c4620, 0x0)
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-aws/aws/resource_aws_elastic_transcoder_preset.go:506 +0x355
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc000466780, 0xc0008cda40, 0x3c8e740, 0xc00055fb00, 0xc0005aa330, 0x0, 0x0)
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.2.0/helper/schema/resource.go:455 +0x119
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ReadResource(0xc00000eb10, 0x54dc6a0, 0xc000ccae40, 0xc0008cd860, 0xc00000eb10, 0xc000ccae40, 0xc000953a80)
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.2.0/internal/helper/plugin/grpc_provider.go:525 +0x3d8
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ReadResource_Handler(0x4b24220, 0xc00000eb10, 0x54dc6a0, 0xc000ccae40, 0xc000826e40, 0x0, 0x54dc6a0, 0xc000ccae40, 0xc0000427e0, 0xd4)
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.2.0/internal/tfplugin5/tfplugin5.pb.go:3153 +0x217
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: google.golang.org/grpc.(*Server).processUnaryRPC(0xc0003ca000, 0x54fb800, 0xc00009db00, 0xc000702900, 0xc00072de90, 0x7e876d0, 0x0, 0x0, 0x0)
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:995 +0x460
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: google.golang.org/grpc.(*Server).handleStream(0xc0003ca000, 0x54fb800, 0xc00009db00, 0xc000702900, 0x0)
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:1275 +0xd97
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc00012e3f0, 0xc0003ca000, 0x54fb800, 0xc00009db00, 0xc000702900)
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:710 +0xbb
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: created by google.golang.org/grpc.(*Server).serveStreams.func1
[DEBUG] plugin.terraform-provider-aws_v2.34.0_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:708 +0xa1
[DEBUG] plugin: plugin process exited: path=/...../.terraform/plugins/linux_amd64/terraform-provider-aws_v2.34.0_x4 pid=11304 error="exit status 2"
```

**Comment by bflad** [comment]
Hi again @robstar 👋 This should have been fixed with https://github.com/terraform-providers/terraform-provider-aws/pull/11944, which was released with version 2.51.0 of the Terraform AWS Provider a few weeks back. I have also submitted https://github.com/terraform-providers/terraform-provider-aws/pull/12612 which verifies that importing a resource without a description should be fine now. 👍
